core/src/main/scala/kafka/log/FileMessageSet.scala

```diff
@@ -225,7 +225,7 @@ class FileMessageSet private[kafka](@volatile var file: File,
       case tl: TransportLayer => tl.transferFrom(channel, position, count)
       case dc => channel.transferTo(position, count, dc)
     }).toInt
-    trace("FileMessageSet " + file.getAbsolutePath + " : bytes transferred : " + bytesTransferred
+    FileMessageSet.trace("FileMessageSet " + file.getAbsolutePath + " : bytes transferred : " + bytesTransferred
```

Comment:
You can add a `import FileMessageSet._` at the top of the class block to avoid having to add the class name before `trace`.